js/app.js

```diff
@@ -103,80 +134,84 @@ for (let i = 0; i < 4; i++) {
-//     break;
-//   } if (userResponse === favorite[7]) {
-//     console.log('That is correct');
-//     score++;
-//     break;
-//   } if (userResponse === favorite[8]) {
-//     console.log('That is correct');
-//     score++;
-//     break;
-//   } if (userResponse === favorite[9]) {
-//     console.log('That is correct');
-//     score++;
-//     break;
-//   } else {
-//     console.log('That is incorrect');
-//   }
-//   //}
-//   console.log(`you have ${moreAttempts - j - 1} attempts remaining`);
-// }
-let answeredCorrectly = false;
-
-for (let i = 0; i < 6; i++) {
-  let userResponse = prompt(`guess one of my top 10 favorite foods! You have ${moreAttempts - i} attempts!`).toLowerCase();
-  for (let j = 0; j < favorite.length; j++) {
-    if (userResponse === favorite[j]) {
-      console.log('That is correct!');
-      answeredCorrectly = true;
-      score++;
-      break;
-    } if (userResponse !== favorite[j]) {
-      console.log('That is incorrect!');
+  let favorite = ['pizza', 'taco', 'cheeseburger', 'lasagna', 'baconeggncheese', 'burrito', 'chickenparm', 'padthai', 'friedchicken', 'pho'];
+
+  console.log(favorite.length);
+
+  let moreAttempts = 6;
+
+  // for (let i = 0; i < 6; i++) {
+  //   //for (let j = 0; j < favorite.length; j++) {
+  //   let userResponse = prompt(`guess one of my top 10 favorite foods! You have ${moreAttempts - i} attempts!`).toLowerCase();
+  //   if (/*favorite.includes(userResponse)*/userResponse === favorite[0] /*|| favorite[1] || favorite[2] || favorite[3] || favorite[4] || favorite[5] || favorite[6] || favorite[7] || favorite[8] || favorite[9]*/) {
+  //     console.log('That is correct');
+  //     score++;
+  //     break;
+  //   } if (userResponse === favorite[1]) {
+  //     console.log('That is correct');
+  //     score++;
+  //     break;
+  //   } if (userResponse === favorite[2]) {
+  //     console.log('That is correct');
+  //     score++;
+  //     break;
+  //   } if (userResponse === favorite[3]) {
+  //     console.log('That is correct');
+  //     score++;
+  //     break;
+  //   } if (userResponse === favorite[4]) {
+  //     console.log('That is correct');
+  //     score++;
+  //     break;
+  //   } if (userResponse === favorite[5]) {
+  //     console.log('That is correct');
+  //     score++;
+  //     break;
+  //   } if (userResponse === favorite[6]) {
+  //     console.log('That is correct');
+  //     score++;
+  //     break;
+  //   } if (userResponse === favorite[7]) {
+  //     console.log('That is correct');
+  //     score++;
+  //     break;
+  //   } if (userResponse === favorite[8]) {
+  //     console.log('That is correct');
+  //     score++;
+  //     break;
+  //   } if (userResponse === favorite[9]) {
+  //     console.log('That is correct');
+  //     score++;
+  //     break;
+  //   } else {
+  //     console.log('That is incorrect');
+  //   }
+  //   //}
+  //   console.log(`you have ${moreAttempts - j - 1} attempts remaining`);
+  // }
```

Comment:
Glad to see the for loop doing the work of all this code! Consider deleting obsolete code that's already commented out.